glslc/test/glslc_test_framework.py

```diff
@@ -290,10 +290,11 @@ def runTest(self):
             test_status = TestStatus(
                 process.returncode, output[0], output[1],
                 self.directory, self.file_shaders)
-            for test_method in get_all_test_methods(self.test.__class__):
-                success, message = getattr(self.test, test_method)(test_status)
-                if not success:
-                    break
+            run_results = [getattr(self.test, m)(test_status)
```

Comment:
Why did "test_method" become "m"?  That's not exactly an enhancement to readability...